xstatic/pkg/font_awesome/__init__.py

```diff
@@ -24,19 +24,19 @@
 KEYWORDS = '%s xstatic' % NAME
 
 # XStatic-* package maintainer:
-MAINTAINER = 'Thomas Waldmann'
-MAINTAINER_EMAIL = 'tw@waldmann-edv.de'
+MAINTAINER = 'Rob Cresswell'
+MAINTAINER_EMAIL = 'robert.cresswell@outlook.com'
 
 # this refers to the project homepage of the stuff we packaged:
-HOMEPAGE = 'http://fortawesome.github.io/Font-Awesome/'
+HOMEPAGE = 'https://github.com/FortAwesome/Font-Awesome'
 
 # this refers to all files:
-LICENSE = '(same as %s)' % DISPLAY_NAME
+LICENSE = 'SIL OFL 1.1'
```

Comment:
they use different licenses for different parts of their project.
so maybe it is easiest to keep the "same as" here.

Reply:
That license file is added, and  js, css, scss, svgs/*  have this in first lines added.
We have also to use a classifier from that list in the toml file

https://pypi.org/classifiers/

Reply:
It is clarified here https://github.com/FortAwesome/Font-Awesome#license

Attribution is required by MIT, SIL OLF, and CC BY licenses. 

arg

Reply:
on test it is build 2 I removed build 1, but it still wants to increment. 


https://test.pypi.org/project/XStatic-Font-Awesome/6.2.1.2/